From ee003999a28841d25d0c0136e981305aae5a7b39 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Mon, 24 Aug 2026 13:38:02 -0300
Subject: [PATCH 38/52] UPSTREAM: powerpc: Fix preprocessor conditional in
 soft-float __longjmp (bug 34530)

Commit 24d188a2a149b9933e17240e454f917e065cdf06 left a stray #endif in
the powerpc32 soft-float __longjmp-common.S. Remove it, matching the
fpu variant.

Checked with a build for powerpc32-linux-gnu-soft.

(cherry picked from commit 5f5bc63a4d8a7eec7f11e56664ca76455d837b22)
---
 sysdeps/powerpc/powerpc32/__longjmp-common.S | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sysdeps/powerpc/powerpc32/__longjmp-common.S b/sysdeps/powerpc/powerpc32/__longjmp-common.S
index 9da86330f7..f1da933704 100644
--- a/sysdeps/powerpc/powerpc32/__longjmp-common.S
+++ b/sysdeps/powerpc/powerpc32/__longjmp-common.S
@@ -37,7 +37,6 @@ ENTRY (__longjmp_symbol)
 	CHECK_SP (r24)
 # endif
 	mr r1,r24
-# endif
 #else
 	lwz r1,(JB_GPR1*4)(r3)
 #endif
-- 
2.55.0

